templates/admin/jqadm/product/item-media.php

```diff
@@ -73,9 +73,21 @@ class="btn btn-card-header act-delete fa" tabindex="<?= $this->get( 'tabindex' )
 								<input class="item-url" type="hidden"
 									v-bind:name="`<?= $enc->js( $this->formparam( ['media', '_idx_', 'media.url'] ) ) ?>`.replace('_idx_', idx)"
 									v-model="item['media.url']">
-								<img v-if="item['media.preview']" class="item-preview"
+								<img v-if="item['media.preview'] && (item['media.mimetype'].startsWith('image') || item['media.mimetype'].startsWith('video'))" class="item-preview"
 									v-bind:src="item['media.preview']"
 									v-bind:alt="item['media.label']">
+								<div v-else-if="item['media.mimetype'] === 'audio/mpeg'" class="audio-player">
```

Comment:
Browsers support more audio types than `audio/mpeg`, aren't they? So `v-if="item['media.preview'] && item['media.mimetype'].startsWith('audio')"` seems to be more appropriate.

Reply:
Causes an error in Vue

Reply:
This is really weird: when using `.startsWith('audio')` or `.substring(0,5) === 'audio'` an error occurs:

![Screenshot 2023-03-09 at 12 53 50](https://user-images.githubusercontent.com/213803/224015730-76498146-6bcd-4cf6-a305-05f681d504aa.png)
Not sure, but it seems this is a TYPO3 problem?

As long as this error exists, let's please use `item['media.mimetype'] === 'audio/mpeg'`.